vignettes/programming_strategy.Rmd

```diff
@@ -317,9 +322,9 @@ This rule is applicable only if both functions are part of `{admiral}`.
 Every function that is exported from the package must have an accompanying header 
 that should be formatted according to the [roxygen2](https://roxygen2.r-lib.org/) convention. 
 
-In addition to the roxygen2 tags, `@keywords` is also used. 
+In addition to the roxygen2 tags, the `@family` and `@keywords` tags are also used. 
 
-The keywords are used to categorize the function. Please see section "Categorization of functions".
+The family/keywords are used to categorize the function. Please see section [Categorization of functions](programming_strategy.html#categorization-of-functions).
```

Comment:
```suggestion
The family/keywords are used to categorize the function, which is used both on our website and the internal package help pages. Please see section [Categorization of functions](programming_strategy.html#categorization-of-functions).
```

Reply:
not sure how to word, but I want folks to know how we use them...

Reply:
I thought thats what the mention of the categorization of functions section is for, it is rather descriptive:

![image](https://github.com/pharmaverse/admiraldev/assets/43018762/5684c191-1fbf-4e5e-900c-40a4d006f90e)

Reply:
that is pretty thorough!  sorry I did not see that.